internal/command/volumes/destroy.go

```diff
@@ -37,26 +38,18 @@ number to operate. This can be found through the volumes list command`
 
 func runDestroy(ctx context.Context) error {
 	var (
-		io       = iostreams.FromContext(ctx)
-		colorize = io.ColorScheme()
-		client   = client.FromContext(ctx).API()
-		volID    = flag.FirstArg(ctx)
+		io     = iostreams.FromContext(ctx)
+		client = client.FromContext(ctx).API()
+		volID  = flag.FirstArg(ctx)
+
+		confirm bool
+		err     error
```

Comment:
no need to declare vars here, in Go you usually do`if confirm, err := ...` and keep going